source/ShipInfoDisplay.cpp

```diff
@@ -37,16 +38,16 @@ using namespace std;
 
 ShipInfoDisplay::ShipInfoDisplay(const Ship &ship, const Depreciation &depreciation, int day)
 {
-	Update(ship, depreciation, day);
+	Update(ship, depreciation, day, true);
 }
 
 
 
-// Call this every time the ship changes.
-void ShipInfoDisplay::Update(const Ship &ship, const Depreciation &depreciation, int day)
+// Call this every time the ship changes. Sale is true for the shipyard panel only.
```

Comment:
```suggestion
// Call this every time the ship changes. Sale is true for the shipyard and outfitter panels only.
```
Isnt this more correct?

Reply:
Yes, you're right.